### PR TITLE
Update french.xml

### PR DESCRIPTION
**Pull request by Need74**
Good morning
I have just completely restarted the translation of Notepad++ into fr Essentially there are many deletions of capital letters in the middle of sentences (non-compliant in French) Replacing the apostrophes ' with ' (alt+0146) (to make it prettier) Replacement of "" (&quot;) by French quotation marks "" (alt+174 & alt175)(to make it prettier too) Some formulations which were not very clear, or even incorrect. I'm still tweaking a few things but that will be for a future PR Sincerely
Dennis

### CODE REVIEW

**Comment by Need74** [comment]
If you want to quickly see the effect of the differences I am attaching the Tatar file. xml
You just have to copy it into the Notepad++-backslash-Localization folder and then choose to change the Tatar language (I hope it's not yours)
Sincerely
Dennis
[tatar.xml.patch](https://github.com/user-attachments/files/15962741/tatar.xml.patch)
You must rename tatar.xml.patch in tatar.xml before copying

File: PowerEditor/installer/nativeLang/french.xml
Comment (donho) [inline PowerEditor/installer/nativeLang/french.xml:1106]: In the latest release (v8.6.8), this entry is enlarged.

**Review by donho — CHANGES_REQUESTED** [review]
Looks great! Please follow the suggestions below:

File: PowerEditor/installer/nativeLang/french.xml
Comment (Need74) [reply PowerEditor/installer/nativeLang/french.xml:225]: So it is better not to talk about "braces" and only put " { } or [ ] or ( ) "
The note is also useful for the English "braces text" and I can put a PR for that.
"Select All Between &amp;Matching Braces" Will be "Select All Between &amp;Matching  { } or [ ] or ( )"

File: PowerEditor/installer/nativeLang/french.xml
Comment (Need74) [reply PowerEditor/installer/nativeLang/french.xml:358]: Thank you
The note is also useful for the English "None (Normal Text)" and I can put in the same PR for that.

**Review by Need74 — COMMENTED** [review]
Just an error " ? "

File: PowerEditor/installer/nativeLang/french.xml
Comment (donho) [reply PowerEditor/installer/nativeLang/french.xml:225]: @Need74 
OK for French to contain "{ } or [ ] or ( )".
For English, I will take care of it myself.

**Review by donho — CHANGES_REQUESTED** [review]
For the PR being accepted, please 
1. remove the modification of english.xml from this PR.
2. Follow the given suggestions about french.xml

**Comment by Need74** [comment]
> For the PR being accepted, please
> 
>     1. remove the modification of english.xml from this PR.
> 
>     2. follow the given suggestions about french.xml

Thank you for taking the time to read and understand the meaning of my PR.
It seems obvious to me that modifying the "English language master" file on which all translations depend must be done with the greatest care.
I don't know if you understood it, but the multiple line breaks I made were intended to make it easier to read the file in text mode and to better identify the beginnings and ends of main sections.
At this time I can no longer intervene on the modification of the english.xml file since the PR is closed and therefore do not understand what you expect from me.
For the PR in progress on french.xml I continue to follow it while waiting for its approval.
I remain at your disposal for any further information and thank you for your involvement in the world of free software.
Sincerely
Dennis

**Comment by Need74** [comment]
> > At this time I can no longer intervene on the modification of the english.xml file since the PR is closed and therefore do not understand what you expect from me.
> 
> You modified english.xml in this PR, too. Look at the Files changed tab.

I hope I did the right thing, it now says: "this commit does not contain any modifications" "the file is deleted"

**Comment by donho** [comment]
@Need74 
No, the action you have done is deleting english.xml. What you should do is restore english.xml as it was.

**Review by Need74 — COMMENTED** [review]
I have reset all

**Comment by Need74** [comment]
> @Need74 No, the action you have done is deleting english.xml. What you should do is restore english.xml as it was.

What should I do to restore the english.xml file to its original state?
When I try to edit the file to restore it, the system tells me that the file does not exist.
I specify that I work exclusively with the github graphical interface because I have never used the command lines.
I will need a point by point explanation to get there.
In any case I can also close my PR and create a new one with only the accepted modifications of my french.xml file.
thank you for your reply

**Review by Need74 — COMMENTED** [review]
After restor

**Review by donho — CHANGES_REQUESTED** [review]
1. Modify`<Item id="43053" name="Sélectionner tout entre les 2 accolades {}"/>` to `<Item id="43053" name="Sélectionner tout entre les 2 accolades {} [] ou ()"/>`
2. Modify `<Item CMDID="10003" name="Déplacer vers une nouvelle instance (fenêtre)"/>` to `<Item CMDID="10003" name="Déplacer vers une nouvelle instance"/>`
3. Modify `<Item CMDID="10004" name="Ouvrir dans une nouvelle instance (fenêtre)"/>` to `<Item CMDID="10004" name="Ouvrir dans une nouvelle instance"/>`
4. Restore `<Item id="1632" name="Dans la sélection"/>`
5. Restore `<Item id="20012" name="Ignorer la casse"/>`
6. Restore `<Item id="6303" name="Taille de l'indentation :"/>`

**Comment by Need74** [comment]
> 1. Modify`<Item id="43053" name="Sélectionner tout entre les 2 accolades {}"/>` to `<Item id="43053" name="Sélectionner tout entre les 2 accolades {} [] ou ()"/>`
> 
>     2. Modify `<Item CMDID="10003" name="Déplacer vers une nouvelle instance (fenêtre)"/>` to `<Item CMDID="10003" name="Déplacer vers une nouvelle instance"/>`
> 
>     3. Modify `<Item CMDID="10004" name="Ouvrir dans une nouvelle instance (fenêtre)"/>` to `<Item CMDID="10004" name="Ouvrir dans une nouvelle instance"/>`
> 
>     4. Restore `<Item id="1632" name="Dans la sélection"/>`
> 
>     5. Restore `<Item id="20012" name="Ignorer la casse"/>`
> 
>     6. Restore `<Item id="6303" name="Taille de l'indentation :"/>`

I hope I have done what is necessary, I still have some improvements to make but this will be the subject of a future PR, when I have fully mastered the new version.
Sincerely

Dennis

**Review by donho — APPROVED** [review]

**Pull request closed without merging**
